cg/services/order_validation_service/models/order.py

```diff
@@ -11,11 +11,11 @@ class Order(BaseModel):
     connect_to_ticket: bool = False
     customer: str = Field(min_length=1)
     delivery_type: DataDelivery
+    order_type: OrderType = Field(alias="project_type")
```

Comment:
Not sure that we want to continue calling this "project type" in the FE and the parser? Should we rename it?

Reply:
I feel like order type has sort of stuck. Would be nice to refer to it that way in the FE as well